monitor/mesos.go

```diff
@@ -79,12 +83,21 @@ func (m *MesosMonitor) getAgentIPAddressFromPID(pid string) string {
 	return strings.Split(tmp, ":")[0]
 }
 
-func (m *MesosMonitor) getTasks() map[string][]mesos.Task {
+func (m *MesosMonitor) getTasks() (map[string][]mesos.Task) {
 
 	tasksMap := map[string][]mesos.Task{}
+
 	tasksResponse, _ := m.mesosConn.GetMesosTasks()
 	for _, task := range tasksResponse.Tasks {
 		if task.State == "TASK_RUNNING" {
+
+			for _, label := range task.Labels {
+				for _, protectedTasksLabel := range m.protectedTasksLabels {
+					if label.Key == protectedTasksLabel && strings.ToUpper(label.Value) == "TRUE" {
```

Comment:
this also can be 
`
task.IsProtected == (protectedTasksLabel && strings.ToUpper(label.Value) == "TRUE")
`